app/helpers/application_helper.rb

```diff
@@ -199,24 +199,8 @@ def url_for_db(db, action = "show", item = nil) # Default action is show
   def view_to_url(view, parent = nil)
     association = view.scoped_association
     # Handle other sub-items of a VM or Host
-    case view.db
-    when "AdvancedSetting"  then association = "advanced_settings"
-    when "CloudNetwork"     then association = "cloud_networks"
-    when "OrchestrationStackOutput" then association = "outputs"
-    when "OrchestrationStackParameter" then association = "parameters"
-    when "OrchestrationStackResource"  then association = "resources"
-    when "Filesystem"       then association = "filesystems"
-    when "FirewallRule"     then association = "firewall_rules"
-    when "GuestApplication" then association = "guest_applications"
-    when "Patch"            then association = "patches"
-    when "RegistryItem"     then association = "registry_items"
-    when "ScanHistory"      then association = "scan_histories"
-    when "SystemService"
-      case parent.class.base_class.to_s.downcase
-      when "host" then association = "host_services"
-      when "vm"   then association = @lastaction
-      end
-    end
+    association = class_to_db(view.db, parent)
```

Comment:
I don't know about the name `class_to_db`...   I don't know what db means.  Is it really an association? If so, maybe view_to_association or something similar would be better.